app/helpers/application_helper.rb

```diff
@@ -16,6 +16,10 @@ def admin_or_moderator?
     access_denied
   end
 
+  def user_not_banned?
```

Comment:
We can verify that user is locked simply by "current_user.banned?".
This helper method (user_not_banned?) was added intentionally to avoid ugly-looking form "!current_user.banned?" for checking if user isn't locked. Mentors added this requirement at the last demo.